lms/djangoapps/mobile_api/course_info/views.py

```diff
@@ -88,6 +88,15 @@ def list(self, request, course, *args, **kwargs):
                     course_id=course.id,
                     static_asset_path=course.static_asset_path
                 )
+                handouts_html = replace_jump_to_id_urls(
+                    handouts_html,
+                    course_id=course.id,
+                    jump_to_id_base_url="/courses/{}/jump_to_id/".format(course.id)
```

Comment:
Part of the refactoring I'm trying to do right now will involve using a function that uses URL reversal.